composer.json

```diff
@@ -26,7 +26,6 @@
         "ext-dom": "*",
         "ext-json": "*",
         "alexkart/curl-builder": "^1.0",
-        "psr/container": "1.0.0",
```

Comment:
Can you use the container implementation from `yiisoft/test-support` instead of `yiisoft/di` in test cases?

Reply:
Yes, I could. Will do.